src/njoy21/input/ACER/Card9/Mte.hpp

```diff
@@ -0,0 +1,16 @@
+struct Mte {
+  using Value_t = int;
+  static std::string name(){ return "mte"; }
+
+  static std::string description() {
+    return
+        "The mte argument is the material number for the thermal elastic \n"
```

Comment:
It is a reaction number, but not strictly an ENDF number. It's one that NJOY uses internally.